live-examples/js-examples/array-tolocalestring.html

```diff
@@ -0,0 +1,9 @@
+<pre>
+<code id="static-js">var array1 = [1, 'a', new Date()];
+
+var string = array1.toLocaleString();
+
+console.log(string);
+// expected output: "1,a,17/01/2018, 15:54:20"
```

Comment:
This output will be both timezone and locale dependent. Could we make timezone and locale fixed using something like:

```
var array1 = [1, 'a', new Date('04 Dec 1995 14:12:00 UTC')];

var localeString = array1.toLocaleString('de', {timeZone: "UTC"});

console.log(localeString);
// expected output: "1,a,4.12.1995, 14:12:00"
```

This had the added benefit of showing the parameters to `toLocaleString`.

Reply:
The timezones and locales are quite broad, so I used 'en' and 'UTC'. Would that be okay?

Reply:
I'm not sure what you mean by "broad". Yes, it's fine to use 'en'. But you're still using `Date()` without any arguments, so the `//expected output:` will not match. I think it would be better to pass a date string to the `Date()` constructor, including the locale, so the date is fixed and will match `expected output`. Like:

```
var array1 = [1, 'a', new Date('04 Dec 1995 14:12:00 UTC')];
```

Actually, thinking about it some more, it would be great to have more than one call to `toLocaleString`, with different locales, so you can see the difference:

```
var array1 = [1, 'a', new Date('04 Dec 1995 14:12:00 UTC')];

var localeString = array1.toLocaleString('en', {timeZone: "UTC"});
console.log(localeString);
// expected output: "1,a,12/4/1995, 2:12:00 PM"

localeString = array1.toLocaleString('de', {timeZone: "UTC"});
console.log(localeString);
// expected output: "1,a,4.12.1995, 14:12:00"
```

Reply:
from _broad_ I meant all the locales - [http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry](url).
We can't add the locales, so I used English ('en')